bokeh/command/subcommands/serve.py

```diff
@@ -727,6 +727,11 @@ class Serve(Subcommand):
         )),
     )
 
+    def _update_server_kwargs(self, args: argparse.Namespace, server_kwargs: Dict[str, Any]) -> None:
```

Comment:
Docstring should not start with a newline to match the rest of the code base. Suggest `customize_kwargs` as a different (and public) name. 

Would it be possible for this to be a configurable hook that can be passed in, rather than a method that has to be overridden in a subclass? 

Might also be preferable to have this method/hook return a new dict rather than mutate its argument.

Reply:
>customize_kwargs as a different (and public) name.

Sounds good.

>Would it be possible for this to be a configurable hook that can be passed in, rather than a method that has to be overridden in a subclass?

I'd certainly be happy if to make that possible but presumably that would have to be loaded from another file or from app_hooks and therefore be a little orthogonal.

>Might also be preferable to have this method/hook return a new dict rather than mutate its argument.

I did that initially but I wanted to make sure that the method does not accidentally override existing kwargs because it doesn't see them, but I can guess I could still pass in the original server_kwargs so the subclass can inspect it before overriding the kwargs.